components/hab/src/cli.rs

```diff
@@ -323,7 +323,7 @@ pub fn get(feature_flags: FeatureFlag) -> App<'static, 'static> {
                     (aliases: &["e", "ex", "exp", "expo", "expor"])
                     (@arg ORIGIN: +required +takes_value {valid_origin})
                     (@arg PAIR_TYPE: -t --type +takes_value {valid_pair_type}
-                        "Export either the 'public' or 'private' key")
+                        "Export either the 'public' or 'secret' key")
```

Comment:
Thanks! I've suggested a change to define 'secret' with the same vocabulary that we're using int the documentation.

```suggestion
                        "Export either the 'public' or 'secret' key. The 'secret' key is the origin private key")
```

Reply:
I've added this in now!